tests/ContaoCommunityAlliance/DcGeneral/Test/DataDefinition/Definition/DefaultPropertiesDefinitionTest.php

```diff
@@ -0,0 +1,283 @@
+<?php
+
+/**
+ * This file is part of contao-community-alliance/dc-general.
+ *
+ * (c) 2013-2017 Contao Community Alliance.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * This project is provided in good faith and hope to be usable by anyone.
+ *
+ * @package    contao-community-alliance/dc-general
+ * @author     Sven Baumann <baumann.sv@gmail.com>
+ * @copyright  2013-2017 Contao Community Alliance.
+ * @license    https://github.com/contao-community-alliance/dc-general/blob/master/LICENSE LGPL-3.0
+ * @filesource
+ */
+
+namespace ContaoCommunityAlliance\DcGeneral\Test\DataDefinition\Definition;
+
+use ContaoCommunityAlliance\DcGeneral\DataDefinition\Definition\DefaultPropertiesDefinition;
+use ContaoCommunityAlliance\DcGeneral\DataDefinition\Definition\Properties\Collection\DefaultSubProperty;
+use ContaoCommunityAlliance\DcGeneral\DataDefinition\Definition\Properties\DefaultProperty;
+use ContaoCommunityAlliance\DcGeneral\DataDefinition\Definition\Properties\DefaultCompoundProperty;
+use ContaoCommunityAlliance\DcGeneral\Test\TestCase;
+
+/**
+ * This tests DefaultPropertiesDefinition
+ *
+ * @coversDefaultClass \ContaoCommunityAlliance\DcGeneral\DataDefinition\Definition\DefaultPropertiesDefinition
+ */
+class DefaultPropertiesDefinitionTest extends TestCase
+{
+    /**
+     * Get the default properties definition.
+     *
+     * @return DefaultPropertiesDefinition
+     */
+    private function getDefintiion()
+    {
+        return new DefaultPropertiesDefinition();
+    }
+
+    /**
+     * Get a default property.
+     *
+     * @param string $name The property name.
+     *
+     * @return DefaultProperty
+     */
+    private function getDefaultProperty($name)
+    {
+        return new DefaultProperty($name);
+    }
+
+    /**
+     * Get a default collection property.
+     *
+     * @param string $name The property name.
+     *
+     * @return DefaultSubProperty
+     */
+    private function getDefaultCollectionProperty($name)
```

Comment:
Has to be adapted to the name `SubProperty`.

Reply:
change